shell/src/main/java/org/apache/zeppelin/shell/ShellInterpreter.java

```diff
@@ -57,17 +57,14 @@ public ShellInterpreter(Properties property) {
 
   @Override
```

Comment:
I think we can move `createSecureConfiguration` from `ShellSecurityImpl` to `ShellInterpreter`, then `ShellSecurityImpl` could be deleted